src/plugins/add-cal/reports/a11y-1-en.hbs

```diff
@@ -0,0 +1,114 @@
+---
+{
+	"title": "Accessibility assessment #1 - Add to Calendar",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "add-cal",
+	"parentdir": "add-cal",
+	"parentPage": "Add to calendar",
+	"parentPageURL": "add-cal",
```

Comment:
There is something that don't work with the page breadcrumb. I do think that is related to a different issue.

FYI - @GormFrank

Reply:
Let fix that later, we will need to review how the template system work for the wet-boew website in the future.